src/posix.mak

```diff
@@ -90,11 +90,12 @@ else
   # Auto-bootstrapping, will download dmd automatically
   # Keep var below in sync with other occurrences of that variable, e.g. in circleci.sh
   HOST_DMD_VER=2.072.2
+  HOST_DMD_YEAR=2016
   HOST_DMD_ROOT=$(TMP)/.host_dmd-$(HOST_DMD_VER)
   # dmd.2.072.2.osx.zip or dmd.2.072.2.linux.tar.xz
   HOST_DMD_BASENAME=dmd.$(HOST_DMD_VER).$(OS)$(if $(filter $(OS),freebsd),-$(MODEL),)
   # http://downloads.dlang.org/releases/2.x/2.072.2/dmd.2.072.2.linux.tar.xz
-  HOST_DMD_URL=http://downloads.dlang.org/releases/2.x/$(HOST_DMD_VER)/$(HOST_DMD_BASENAME)
+  HOST_DMD_URL=https://s3-us-west-2.amazonaws.com/downloads.dlang.org/releases/$(HOST_DMD_YEAR)/$(HOST_DMD_BASENAME)
```

Comment:
> It would be much better if we could avoid exposing "implementation details" like the real download server, and use dlang.org links.

Well, we could have downloads.dlang.io where the year mapping is hard-coded, but then all traffic would flow to a single point of failure and the entire point of using S3 is gone.

>  In fact, why not move the HTML pages of downloads.dlang.org to the dlang.org server?

Because they are automatically generated and I bet @MartinNowak wouldn't like sending yet another PR on every update.